kaldb/src/main/java/com/slack/kaldb/chunkManager/IndexingChunkManager.java

```diff
@@ -231,6 +234,7 @@ public void onFailure(Throwable t) {
    * When the ChunkManager is being closed, roll over active chunk so we can be sure that it's data is persisted in
    * a remote store.
    */
+  @VisibleForTesting
```

Comment:
This annotation should be removed, as recovery nodes will need to manually rollover the active chunk as well.

Reply:
The recovery node work is already in progress, so this would have to be immediately removed. Instead of adding it to just remove it later this week let's not include it here. 

The current expectation is that this method can and will be called externally, so we shouldn't introduce an annotation that indicates otherwise.